crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -281,11 +282,9 @@ impl Schedule {
         label: impl StageLabel,
         func: F,
     ) -> &mut Self {
-        let stage = self.get_stage_mut::<T>(&label).unwrap_or_else(move || {
-            panic!(
-                "stage '{:?}' does not exist or is the wrong type",
-                label.as_label()
-            )
+        let label = label.as_label();
```

Comment:
nitpick: Can you rename the parameter to stage_label. `label = label.as_label()` is a bit confusing.
Same for the `get_stage` and `get_stage_mut` methods below.